src/PackageDependencyViewExtension/PackageDependencyView.xaml

```diff
@@ -86,5 +86,51 @@
                 </TextBlock>
             </Button>
         </Grid>-->
+
+        <!-- Template for the feedback banner -->
+        <DockPanel Name="Feedback" Margin="0,800,0,40" Grid.Row="1">
+            <StatusBar Background="CornflowerBlue" DockPanel.Dock="Bottom" Height="55">
+                <!-- Preview text -->
+                <StatusBarItem>
+                    <Border Height="50">
+                        <TextBlock TextWrapping="Wrap" Padding="10,0,10,0" FontSize="15" Foreground="White" Text="This Feature is in Preview!" VerticalAlignment="Center"/>
+                    </Border>
+                </StatusBarItem>
+                <StatusBarItem Width="193" Margin="100,0,0,0" Height="40">
+                    <Border Name="FeedbackBorder" BorderThickness="1">
+                        <Border.Style>
+                            <Style>
+                                <Style.Triggers>
+                                    <Trigger Property="Border.IsMouseOver" Value="True">
+                                        <Setter Property="Border.BorderBrush" Value="White" />
+                                    </Trigger>
+                                    <Trigger Property="Border.IsMouseOver" Value="False">
+                                        <Setter Property="Border.BorderBrush" Value="#373737" />
+                                    </Trigger>
+                                </Style.Triggers>
+                            </Style>
+                        </Border.Style>
+                        <!-- Feedback button -->
+                        <Button HorizontalAlignment="Right"
+                            Width="150" Click="ProvideFeedback" Background="#373737" Height="30">
+                            <TextBlock TextWrapping="Wrap" FontSize="15" Foreground="White" Height="20" Text="Provide Feedback" VerticalAlignment="Center"/>
```

Comment:
@reddyashish - Aaron reminded me - I think this needs to come from a resx dictionary.

Reply:
Yes, please.

Reply:
Added this to resource file.